lib/components/segmented-control/segment.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import clsx from "clsx";
+import { Text } from "../Typography";
+import * as quillIcons from "@deriv/quill-icons";
+import { SegmentedControlProps } from "./base";
+
+interface SegmentProps {
+    className?: string;
+    icon?: string | React.ReactNode;
+    isDisabled?: boolean;
+    isSelected?: boolean;
+    label?: string;
+    onClick?: (event: React.MouseEvent<HTMLDivElement>) => void;
```

Comment:
Are we gonna support onKeyDown? I'm asking, because in my case I have figma design for focused elements (=== tab). That pushed me to thoughts that I should also add support for keyboard events (on 'Enter' or 'Space')

Reply:
Please, ignore if it's not relevant

Reply:
for now, we don't any requirements or specification regarding this.

Reply:
good point, i'll add that👍🏻